### ISSUE TITLE
About the edge

### ISSUE THREAD

**Issue by suzhenghang** [akirasosa/mobile-semantic-segmentation#10]
Hi @akirasosa ,
         Your result looks good. How did you process the edge? Bilinear interpolation will smooth the detail of edge, right?

**Comment by akirasosa** [comment]
@suzhenghang 
Yes. If you use nearest neighbor as interpolation, the edge will be rough. I have already tried it.

**Comment by suzhenghang** [comment]
@akirasosa 
Thanks. By the way, have you compare the Dice Loss with Softmax Cross Entropy Loss ?

**Comment by akirasosa** [comment]
@suzhenghang 
I monitor binary cross entropy. I can see a little difference between their behaviour. Though I have not tried to use binary cross entropy as a loss, it may be interesting to try.

**Comment by suzhenghang** [comment]
@akirasosa ,
       How do you get the final predicted mask from the output of the last convolutional layer after sigmoid ?

**Comment by akirasosa** [comment]
@suzhenghang 
sigmoid itself is a mask.

**Comment by suzhenghang** [comment]
@akirasosa ,
      yep, will you set a Threshold to the output of the sigmoid to set the value to 0 or 1
